### ISSUE TITLE
AntiAdblock on TvTropes

### ISSUE THREAD

**Issue by Juni221** [uBlockOrigin/uAssets#2904]
### URL(s) where the issue occurs

`[https://tvtropes.org/]`

### Describe the issue

Immediately after the page finishes loading, it brings up this text and background, with no way to click past it or anything 

### Screenshot(s)

[https://imgur.com/a/32sUb6A]

### Versions

- Browser/version: [Version 49.0.2623.112 m]
- uBlock Origin version: [uBlock Origin v1.16.12]

### Settings

- [List here all the changes you made to uBO's default settings]

### Notes

[Adding these URLs to "My Filters" seems to work
https://static.tvtropes.org/design/js/google-adblock.js
https://static.tvtropes.org/design/js/ads.js?rev=1531609070
https://secure.quantserve.com/quant.js]

**Comment by okiehsch** [comment]
This is already fixed with a default setup and using a somewhat newer version of Chrome ;).
The `script:inject` filter probably does not work with Chrome 49.
`https://static.tvtropes.org/design/js/google-adblock.js` is not even called on my end because of the existing `script:inject` filter.
Adding
`||static.tvtropes.org/design/js/google-adblock.js$script`
to your filter list should fix the issue.
I lean towards not adding filters to fix an issue for an obsolete browser.
@gorhill what do you thiink about such an issue?

**Comment by uBlock-user** [comment]
They seemed to have rmeoved IL code now. 

> https://static.tvtropes.org/design/js/google-adblock.js is not even called on my end because of the existing script:inject filter.

It is called on my end on Chromium 69, but the script is blocked, so I don't see any anti adblock notice or banner or anything.

**Comment by okiehsch** [comment]
You are right, the script is called if I use chromium, what is blocking it at your end?

I will add
`||static.tvtropes.org/design/js/google-adblock.js$script`
can't hurt and the script is actually called with an up to date browser.

**Comment by uBlock-user** [comment]
> what is blocking it at your end?

by my own filter, `adblock$script,important`
